pyocd/probe/picoprobe.py

```diff
@@ -101,13 +102,10 @@ def close(self):
         self._rd_ep = None
 
     @classmethod
-    def enumerate_picoprobes(cls, uid=None):
+    def enumerate_picoprobes(cls, uid=None) -> List["PicoLink"]:
```

Comment:
I must admit this was new to me. Oh well, more PEPs to read...

Reply:
Yeah. It's all pretty new to me, too. 😄 As long as I've been using python, I haven't been able to use any of the Python 3 features because Python 2 compatibility was required in everything I worked on. So I'm going to start trying to add some type annotations to pyocd to better document the APIs.

And if you think type annotations are fun, check out [PEP 638](https://www.python.org/dev/peps/pep-0638/). 😁